programmers/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,12 @@
+function solution(numbers) {
+    let answer = [];
+    for (let i = 0; i < numbers.length + 1; i++) {
+        for (let j = i + 1; j < numbers.length; j++) {
+            let result = numbers[i] + numbers[j];
+            answer.push(result);
+        }
+    }
+    answer = [...new Set(answer)];
+    answer.sort((a, b) => a - b);
+    return answer;
+}
```

Comment:
1. Set 객체를 사용해서 중복 값을 제거해준 것은 좋으나, 미리 선언해줬으면 더 좋았을 것 같음. 미리 선언하고 add를 통해 Set 객체에 값을 추가하면 됨.
2. 그리고 최대한 반복문 안에서 변수 선언하는건 피해주는 것이 좋음.
3. Set 객체를 배열로 변환 하는 방법은 Array.from()을 사용하면 됨
4. 그리고 굳이 answer을 return할 필요없이 바로 리턴해주는게 더 보기 좋을 것 같음
```suggestion
function solution(numbers) {
    const answer = new Set();
    let result = 0;
    
    for (let i = 0; i < numbers.length + 1; i++) {
        for (let j = i + 1; j < numbers.length; j++) {
            result = numbers[i] + numbers[j];
            answer.add(result);
        }
    }
    return Array.from(answer).sort((a, b) => a - b);
}
```